subprojects/docs/src/samples/build-organization/cross-project-output-sharing/tests/check.sample.conf

```diff
@@ -0,0 +1,4 @@
+commands: [{
+    executable: gradle
+    args: showFile
```

Comment:
❌ This should also test the expected output in order to be useful

Reply:
Added, thanks: https://github.com/gradle/gradle/pull/19519/commits/eba8f2f85bb012f37e5243f310251be9103fe994